packages/web/components/complex/all-pools-table.tsx

```diff
@@ -0,0 +1,707 @@
+    const account = accountStore.getAccount(chainId);
+    const fiat = priceStore.getFiatCurrency(priceStore.defaultVsCurrency)!;
+    const queryActiveGauges = queriesExternalStore.queryActiveGauges;
+
+    const allPools = queriesOsmosis.queryGammPools.getAllPools();
+
+    const allPoolsWithMetrics: PoolWithMetrics[] = useMemo(
+      () =>
+        allPools.map((pool) => {
+          const poolTvl = pool.computeTotalValueLocked(priceStore);
+          const myLiquidity = poolTvl.mul(
+            queriesOsmosis.queryGammPoolShare.getAllGammShareRatio(
+              account.bech32Address,
+              pool.id
+            )
+          );
+
+          return {
+            pool,
+            ...queriesExternalStore.queryGammPoolFeeMetrics.getPoolFeesMetrics(
+              pool.id,
+              priceStore
+            ),
+            liquidity: pool.computeTotalValueLocked(priceStore),
+            myLiquidity,
+            myAvailableLiquidity: myLiquidity.toDec().isZero()
+              ? new PricePretty(fiat, 0)
+              : poolTvl.mul(
+                  queriesOsmosis.queryGammPoolShare
+                    .getAvailableGammShare(account.bech32Address, pool.id)
+                    .quo(pool.totalShare)
+                ),
+            poolName: pool.poolAssets
+              .map((asset) => asset.amount.currency.coinDenom)
+              .join("/"),
+            networkNames: pool.poolAssets
+              .map(
+                (asset) =>
+                  chainStore.getChainFromCurrency(asset.amount.denom)
+                    ?.chainName ?? ""
+              )
+              .join(" "),
+            apr: queriesOsmosis.queryIncentivizedPools
+              .computeMostApr(pool.id, priceStore)
+              .add(
+                // swap fees
+                queriesExternalStore.queryGammPoolFeeMetrics.get7dPoolFeeApr(
+                  pool,
+                  priceStore
+                )
+              )
+              .add(
+                // superfluid apr
+                queriesOsmosis.querySuperfluidPools.isSuperfluidPool(pool.id)
+                  ? new RatePretty(
+                      queriesStore
+                        .get(chainId)
+                        .cosmos.queryInflation.inflation.mul(
+                          queriesOsmosis.querySuperfluidOsmoEquivalent.estimatePoolAPROsmoEquivalentMultiplier(
+                            pool.id
+                          )
+                        )
+                        .moveDecimalPointLeft(2)
+                    )
+                  : new Dec(0)
+              )
+              .maxDecimals(0),
+          };
+        }),
+      [
+        allPools,
+        priceStore,
+        queriesOsmosis.queryGammPoolShare,
+        queriesOsmosis.queryIncentivizedPools,
+        queriesOsmosis.querySuperfluidPools,
+        queriesOsmosis.querySuperfluidOsmoEquivalent,
+        account.bech32Address,
+        queriesExternalStore.queryGammPoolFeeMetrics,
```

Comment:
Not sure if you removed them, but we do actually want the response members of these query stores, since that is what is updated when queries land.

Reply:
I put back the dependencies that were there before, minus some objects that weren't referenced at all within the useMemo hook.